app/views/content/landing/mailing-list-1.md

```diff
@@ -6,7 +6,6 @@ content:
     - content/landing/mailing-list-1/mailing-list
 image: "static/images/content/hero-images/english.jpg"
 layout: "layouts/minimal"
-talk_to_us: false
```

Comment:
Same issue re lack of gap above footer
![image](https://github.com/user-attachments/assets/d2bbf211-dc0c-40fb-b7d5-22775909c786)